tests/iscsi/iscsi_server.pm

```diff
@@ -54,10 +55,20 @@ sub run {
     wait_still_screen(2, 10);
     type_string '132';
     wait_still_screen(2, 10);
-    send_key 'alt-u';                                                                   # un-check use authentication
+    if (is_sle('>=15')) {
```

Comment:
```
my $iscsi_hotkey = is_sle('>=15') ?  'alt-l' : 'alt-u';
send_key $iscsi_hotkey;
```
same logic but more readable format

Reply:
probably you can give more meaningful name to variable because I am not fully aware about test flow so it was just a guess

Reply:
These two send_keys in the if are also not really related to each other.
There are two checkboxes on that config page and for our test both must be unchecked.
On SLE15 the first one is checked by default and on older SLE the second one is checked by default.
So I need to uncheck the right one.
But I think, the code is better readable in the current state, as each send_key has its individual comment that describes, which checkbox is unchecked.